### ISSUE TITLE
List games without results; asking for help

### ISSUE THREAD

**Issue by matttrenary** [matttrenary/rugby-ratings#5]

**Comment by matttrenary** [comment]
Agreed, could even start with a simple message on the front page and a simple contact box/form or email.

**Comment by grjanke** [comment]
We've placed a form on the homepage and linked to it throughout the site. This seems to work for now.

**Comment by matttrenary** [comment]
Discussed and decided a listing of needed scores could be displayed below a score submission form on a standalone "Submit Score" page.
